README.md

```diff
@@ -59,4 +59,28 @@ This will generate the test files into the `api-specification` package. Now the
 npm run docs
 ```
 
-inside the `api-specification` package.
\ No newline at end of file
+inside the `api-specification` package.
+
+### Release
+
+To release the packages to npm, install lerna globally and run
+
+```
+lerna publish --independent
+```
+
+and follow the instructions. This will create a publish commit, which can then be pushed and merged. To push the tags created by lerna, use
+
+```
+git push <remote> --tags
+```
+
+If lerna has errors about git tags, they may need to be deleted using this command.
+
+```
+git tag | xargs git tag -d
+```
+
+and rerun the publish command.
+
+_Note that you will need to be logged into npm on the command line, as well as having the required permission to push to the npm repository for publish to succeed._
```

Comment:
Do we know what it required to get the npm permissions? Do you need to be the `owner` of the project, or we just need Colin to grant us permissions to the npm projects?